doc/en/source/https.rst

```diff
@@ -187,3 +187,334 @@ FogFlow dashboard can be opened in web browser to see the current system status
 
 	If self-signed SSL certificate is being used, a browser warning indication can be seen that the crtificate should not be trusted.
 	It can be proceeded past this warning to view the FogFlow dashboard web page via https.
+
+
+
+Secure FogFlow using Identity Management
+==========================================
+
+Identity management(IdM) is a process for identifying, authenticating individuals or groups to have access to applications or system by associating some auth token with established identities. IdM is the task of controlling data about users or applications.
+
+
+Terminology
+---------------
+
+**Keyrock** is the FIWARE component responsible for Identity Management. Keyrock also provide feature to add OAuth2-based authentication and authorization security in order to secure services and applications.
+
+**PEP Proxy Wilma** is a FIWARE Generic Enabler that enhances the performance of Identity Management. It combines with Keyrock to secure access to endpoints exposed by FIWARE Generic Enablers. Wilma listens for any request, authenticates it from Keyrock and stores it in its cache for a limited period of time. If a new request arrives, Wilma will first check in its cache and if any grant is stored, it will directly authenticate otherwise it will send the request to Keyrock for authentication. 
+
+
+.. figure:: figures/security_architecture.png
+
+
+
+Note: In above diagram n ( n is numeric) is used for cloud node and n’ for edge node.
+
+
+FogFlow cloud node flow:
+
+1. As in architecture diagram, PEP Proxy will register itself first on Keyrock. Detail explanation are is given in below topics of this tutorial.
+
+2. User can access the application directly by using the access-token of PEP proxy.
```

Comment:
[Major] this is generic but it is tested only for designer and applicable only for designer(dashboard). So better to explain only about designer.